### PR TITLE
Fixes for Delegator.Confirm

### PR DESCRIPTION
**Pull request by davywentworth**
- Hitting the 'enter' key no longer repeatedly opens dialogs on top of each other
  - 'Ok' button is now focused

### CODE REVIEW

File: js/Source/Delegators/Delegator.Confirm.js [hunk 1]
```diff
@@ -78,8 +78,13 @@ name: Delegator.Confirm
...
           }
         }
+        var existingPopup = link.retrieve('Bootstrap.Popup');
+        if (existingPopup && existingPopup.visible){
+          existingPopup.show();
+          return existingPopup;
+        }
 
-        var prompt = make_prompt({
+        var prompt = makePrompt({
```
Comment (anutron) [inline js/Source/Delegators/Delegator.Confirm.js]: thanx!

File: js/Source/Delegators/Delegator.Confirm.js [hunk 2]
```diff
@@ -89,7 +94,12 @@ name: Delegator.Confirm
           deleting: (link.get('data-method')||"").toLowerCase() == 'delete'
         }).addClass('hide');
         prompt.inject(document.body);
-        var popup = new Bootstrap.Popup(prompt, {persist: false});
+        var popup = new Bootstrap.Popup(prompt, {
+          persist: false,
+          onShow: function(){
+            this.element.getElement('.btn-ok').focus();
+          },
```
Comment (anutron) [inline js/Source/Delegators/Delegator.Confirm.js]: nix trailing comma

**Comment by anutron** [comment]
This is good stuff, but given that it's going into Behavior, we should fix the padding problem from that screenshot. I.e. the button we create shouldn't have the margin or whatever that buttons get by default. It means that every place we use confirm on a form it'll look broken like that.

**Comment by davywentworth** [comment]
As discussed, the button 'padding' problem is resolved by only making the 'ok' button a button tag.

**Pull request closed without merging**